frontend/src/lib/components/Subscriptions/utils.ts

```diff
@@ -73,23 +73,3 @@ export const timeOptions: LemonSelectOptions = range(0, 24).reduce(
     }),
     {}
 )
-
```

Comment:
Love that this moved off a utils module and into a meaningful place on the model 💖